fe/fe-core/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -1396,4 +1396,10 @@
      */
     @ConfField(mutable = true, masterOnly = true)
     public static int max_dynamic_partition_num = 500;
+
+    /*
+     * Control the max num of backup job per db
+     */
+    @ConfField(mutable = true, masterOnly = true)
+    public static int max_backup_job_num_per_db = 100;
```

Comment:
```suggestion
    public static int max_backup_restore_job_num_per_db = 100;
```
And I think 100 by default is too large. How about 10?

Reply:
done